### PR TITLE
enable external etcd for e2e and update readme

### PR DESCRIPTION
**Pull request by yanmxa**
Signed-off-by: myan <myan@redhat.com>
Referrence: https://github.com/stolostron/multicluster-controlplane/issues/4

### CODE REVIEW

**Review by clyang82 — CHANGES_REQUESTED** [review]
Please update the readme - https://github.com/open-cluster-management-io/multicluster-controlplane/blob/ca2859f567adb6cbe8f06cdc1c7a05eee48dda5d/README.md

File: README.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
 # Get started
 
+## Difference with [multicluster-controlplane](https://github.com/open-cluster-management-io/multicluster-controlplane) in open-cluster-management
+
+This reposotiy is an extension to [open-cluster-management-io/multicluster-controlplane](https://github.com/open-cluster-management-io/multicluster-controlplane) with the following enhancements:
+1. Added managed cluster info addon
+2. Added policy addon
+3. Added managed service account addon
+4. The above addon components are covered in Integration/E2E test cases
```
Comment (clyang82) [inline README.md:9]: maybe remove 4. Here we highlight features in this repo. the test cases is for guarantee the quality. what do you think?

File: README.md [hunk 2]
```diff
@@ -18,4 +31,7 @@ $ make setup-integration
 $ make setup-e2e
 ```
 
+> **Warning**
```
Comment (clyang82) [inline README.md:34]: Add Option 3 for external etcd.

**Review by clyang82 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (yanmxa) [reply README.md]: If this is about highlight features. It's reasonable to remove 4. 

I have removed it from here.

File: README.md (hunk 2 above)
Comment (yanmxa) [reply README.md:49]: I have added the `Option 3:  Run the multicluster-controlplane binary with external etcd` in README.md

In `Option 2` we use external etcd by default. So `Option 3` mainly describes the ability to specify an external etcd when starting a binary server. Starting a binary etcd is described in many [posts](https://medium.com/nirman-tech-blog/setting-up-etcd-cluster-with-tls-authentication-enabled-49c44e4151bb), so I won't go into detail.

**Comment by yanmxa** [comment]
@clyang82 
As you said, I updated the installation of readme to the following module:
1. start controlplane with embedded etcd
2. start controlplane with external etcd
3. start controlplane in local

**Comment by clyang82** [comment]
Perfect. Thanks !
/LGTM

**Pull request merged**